plugins/modules/ntnx_protection_rules.py

```diff
@@ -679,6 +679,10 @@ def check_rule_idempotency(rule_spec, update_spec):
         ):
             return False
 
+    #check if start_time has been updated
+    if rule_spec["spec"]["resources"]["start_time"] != update_spec["spec"]["resources"]["start_time"]:
```

Comment:
This can fail if `start_time` is missing from update or current spec ?

Reply:
possibly true. I didn't check that state, but it definitely ignores the start_time as it is now. We were trying to use this to update the start time to a standard time based on the name of the policy but it passes as nothing to update even though the start time does not match.

Reply:
The usecase is valid. Can you fix this for case if `start_time` is not present in resources ?

Reply:
updated to use .get("start_time") instead of ["start_time"] so that if it is undefined on either side it defaults to None and will be handled by the !=